src/rebar3_hex.erl

```diff
@@ -62,7 +62,13 @@ task_state(State) ->
      case rebar3_hex_config:repo(State) of
          {ok, Repo} -> 
              Opts = get_args(State),
-             {ok, #{args => Opts, repo => Repo, state => State}};
+             case rebar_state:project_apps(State) of 
+                 [App] ->
+                     State1 = rebar_state:current_app(State, App),
```

Comment:
This is called `multi_project` vs say `is_release` because a release might have one sub-app to start with, thus we don't know if we're in a release or not, further would not be necessary to make that distinction in regards to deciding whether to give the user the option of publishing multiple apps or simply going with the one app found.

Reply:
Yes, it should not say anything about `release`. It is unrelated to anything release related.

I think `multi_app` would be better. You are in a single rebar3 project which may have multiple apps, but not multiple projects.

Reply:
@tsloughter, I think the confusion comes from the fact that `rebar3 new` has `release` and `umbrella` for basically the same time (probably even the same thing).

Reply:
Yea, I dislike that. If it was possible to treat `rebar.config` as writable I'd 100% want to remove the `relx` section from `rebar.config` when generated for `rebar3 new umbrella` and would make `rebar3 new release` simply add a `relx` section to the current project's `rebar.config` and create the `config/` files.

I so often have a single app project (so with no `apps/` dir) that builds a release, but it is manual to go from a `rebar3 new app` to a rebar3 project that can build a release.

So yea, agree, probably should never have put the release stuff in `umbrella` template. I think there is a major documentation page missing that can go in detail into each of these.

Reply:
> Yes, it should not say anything about `release`. It is unrelated to anything release related.
> 
> I think `multi_app` would be better. You are in a single rebar3 project which may have multiple apps, but not multiple projects.

Yup, like `multi_app` better 👍 ... and it's shorter 😁